lapack/gonum/dlassq.go

```diff
@@ -6,10 +6,9 @@ package gonum
 
 import "math"
 
-// Dlassq updates a sum of squares in scaled form. The input parameters scale and
-// sumsq represent the current scale and total sum of squares. These values are
-// updated with the information in the first n elements of the vector specified
-// by x and incX.
+// Dlassq updates a sum of squares represented in scaled form. Dlassq returns
+// the values scl and smsq such that
+//  scl^2*smsq = X[0]^2 + ... + X[n-1]^2 + scale^2*sumsq
 //
 // Dlassq is an internal routine. It is exported for testing purposes.
 func (impl Implementation) Dlassq(n int, x []float64, incx int, scale float64, sumsq float64) (scl, smsq float64) {
```

Comment:
Can you note that the implementation is as provided in the supplemental material of https://doi.org/10.1145/3061665?

Reply:
Done.